airbyte-integrations/connectors/destination-exasol/target/id_rsa.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAAgQDndR9nviw2qpN52bIke+CuNwz3P6xMcTcK2Ggc6DTKpD372f4ukxhYgU3EPJXeklNnydgG3+WfNWQgwtwxArQYWiCZyeIjIX4Df3qEfr0VSPSuFYeCuyD2LWGdPYTP1LR09kyYtCs/Lgu9i3FJ3tc19gwkvtXsOBJZURV3+U4zHw== 
```

Comment:
Was this committed by mistake?

Reply:
reverted as well

Reply:
Are these secrets?  They are in github history now, and need to be rolled...

Reply:
they're all test-generated. I'm not sure how to remove them from the repo altogether. close the PR and delete the branch?

Reply:
Then it doesn't matter!

Reply:
As a note, since we have so many popular forks, there actually is no functional way to remove our git history